airflow/providers/snowflake/hooks/snowflake_sql_api.py

```diff
@@ -254,3 +246,30 @@ def get_sql_api_query_status(self, query_id: str) -> dict[str, str | list[str]]:
             }
         else:
             return {"status": "error", "message": resp["message"]}
+
+    def get_sql_api_query_status(self, query_id: str) -> dict[str, str | list[str]]:
+        """
+        Based on the query id async HTTP request is made to snowflake SQL API and return response.
+
+        :param query_id: statement handle id for the individual statements.
+        """
+        self.log.info("Retrieving status for query id %s", {query_id})
```

Comment:
```suggestion
        self.log.info("Retrieving status for query id %s", query_id)
```

The extra braces seem misplaced. I know they existed previously but this seems like a good chance to clean them up.